mkosi.images/minimal-base/mkosi.conf

```diff
@@ -13,6 +13,7 @@ Packages=
         bash
         coreutils
         grep
+        systemd
```

Comment:
what's this for?

Reply:
won't this make the minimal image not-so-minimal anymore?

Reply:
I wonder if we can script it? Some beautiful horrid bash hack

Reply:
Do we care? I don't think the size matters anymore.

Reply:
Well it's slower to build, slower to copy, slower to run - and look at this pretty:

`systemd-run -p NotifyAccess=all --wait --pipe --service-type=notify sh -c 'echo READY=1 | nc -uU -N -q 1 $NOTIFY_SOCKET'`

Reply:
the package on the debian/ubuntu side is netcat-openbsd

Reply:
this also means we get some coverage of an external tool sending notify messages, which is also a nice plus

Reply:
Doesn't work, we use ncat, not netcat-openbsd, get me bash one liner using ncat and we can change it.

Reply:
Or this one? https://packages.debian.org/trixie/ncat but it has many more dependencies

Reply:
With ncat this works: `echo READY=1 | ncat -U --udp $NOTIFY_SOCKET`
But we should use netcat-openbsd, as it only depends on libc and libbsd

Reply:
[root@H ~]# systemd-run --wait --service-type=notify --pty sh -c 'echo READY=1 | ncat -U --udp $NOTIFY_SOCKET'
Running as unit: run-p2364-i2664.service
Press ^] three times within 1s to disconnect TTY.

Finished with result: protocol
Main processes terminated with: code=exited, status=0/SUCCESS
Service runtime: 44ms
CPU time consumed: 38ms
Memory peak: 2.4M (swap: 0B)

Fails with protocol result so not working it seems

Reply:
I look forward to reviewing your PR to switch us to netcat-openbsd